model.py

```diff
@@ -10321,7 +10325,7 @@ class ExternalIntegration(Base, HasFullTableCache):
     CONTENT_SERVER = u'Content Server'
 
     # Integrations with STORAGE_GOAL
-    S3 = u'S3'
+    S3 = u'Amazon S3'
```

Comment:
It's simpler if this is consistent with the NAME used by S3Uploader. There are no existing S3 integrations, so changing this shouldn't affect any real sites.